docs/platform/concepts/projects_accounts_access.rst

```diff
@@ -11,32 +11,40 @@ Organizations and organizational units
 
 Organizations and organizational units are collections of projects. When you sign up to Aiven, an organization is created for you.
 
-You can use these to create a hierarchical structure that fits your needs. Organizational units can be nested within an organization, adding another level to group your projects. This gives you greater flexibility to organize your setup to meet your specific use cases. For example, you can easily split production and testing workloads into different organizational units that are in the same organization. 
+You can use your organization to create a hierarchical structure that fits your needs. Organizational units can be nested within an organization, adding another level to group your projects. This gives you greater flexibility to organize your infrastructure based on your specific use cases. For example, you can easily split production and testing workloads into different organizational units. 
 
 Grouping your projects in organizations and organizational units lets you centrally manage settings like:
 
-* Authentication methods - Only available on the organization level
+* Authentication methods: Only available on the organization level
 
-* ACLs - Can be set on all levels (organization, organizational unit, and project)
+* Access control lists (ACLs): Can be set on all levels (organization, organizational unit, and project)
 
   * ACLs for service plans are inherited, meaning all projects within an organization or organizational unit will have the same service plan.
 
-* Groups - User groups managed at the organization level and assigned to projects
+* Groups: Managed at the organization level and assigned to projects
 
-* Teams - Specific to a single organization or organizational unit and cannot be shared between them
+* Teams: Specific to a single organization or organizational unit and cannot be shared between them
```

Comment:
Shall we add here that this is deprecated?

Reply:
Or delete it completely?